test/Microsoft.AspNet.Mvc.TagHelpers.Test/ImageTagHelperTest.cs

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNet.FileProviders;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Http.Internal;
+using Microsoft.AspNet.Mvc.ModelBinding;
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.Caching;
+using Microsoft.Framework.Caching.Memory;
+using Microsoft.Framework.Runtime;
+
+using Moq;
+using Xunit;
```

Comment:
Please remove and sort `using`s.  That should also clean up the extra blank line.